.changelog/1344.changed.txt

```diff
@@ -0,0 +1 @@
+chore: update Go to 1.21
```

Comment:
These two changelog entries are different, I don't suppose you want to create two separate ones? 😅 
```suggestion
chore: update Go to `v1.21`
```